Dockerfile

```diff
@@ -0,0 +1,7 @@
+FROM openjdk:8-alpine
```

Comment:
We should still use `mvn spring-boot:build-image` instead of adding a dockerfile